modules/ROOT/pages/customize.adoc

```diff
@@ -0,0 +1,107 @@
+is always present and is always the first item. The following properties can be customized:
+** *Name*
+** *Type*
+** *Page*
+** *Publicly available*
+* https://help.salesforce.com/articleView?id=rss_cms_collection.htm&type=5[CMS Collection, role=external, window=blank]
+* *API Carousel*: Displays APIs and API versions that are available to the community from Anypoint Exchange.
+* https://help.salesforce.com/articleView?id=community_builder_rich_text_component.htm&type=5[Rich text editor, role=external, window=blank]
+* https://help.salesforce.com/articleView?id=rss_html_editor.htm&type=5[HTML editor, role=external, window=blank]
+
+Components that are typically used in *API details* pages include:
+
+* *API Header*: Specifies the API name and description.
+* *Tabs*: Contains the following components: *API Documentation Viewer*, which displays API Exchange information,
+*API Console*, which renders information from the API RAML specification, and *Feed*, which displays comments,
+discussions, and questions.
+* *API Access Requestor*: Registers a client application and requests access to the API.
+
+Components that are typically used in *My Application* pages include:
+
+* *Application Listing*: Lists all client applications that the member has registered and can access.
+
+To reorder items on a page, simply drag and drop them.
+
+== Theme
+
+Select the *Theme* paintbrush icon in the top left navigation to access the following general branding parameters:
+
+* *Colors*: These settings include the site's *General* color scheme used throughout the community,
+including paragraph, heading, and button text.
+* *Images*: Specify the site's *Company Logo* and the login page's *Background Image*.
+* *Fonts*: Specify the site's fonts, which include a *Primary Font* and *Header Fonts*. You can also set the
+site's *Text Case*.
+* https://help.salesforce.com/articleView?id=community_builder_theme_adjust.htm&type=5[Theme Settings, role=external, window=blank]
+
+You can also edit CSS rules to adjust your community styling by selecting the *Theme* drop-down list arrow
+located next to the *Theme* title and selecting *Edit CSS*.
+
+Refer to https://help.salesforce.com/articleView?id=community_builder_theme_customize.htm&type=5[Customize Your Community Theme, role=external, window=blank].
+
+== Pages
+
+Select the *Pages* icon, which resembles a bulleted list, in the top left navigation to display the name of the current
+page or select another page in the community to view and edit. The *Pages* menu also offers a *New Page* option.
+
+A new page can be an *Object Page* or *Standard Page*. An *Object Page* is only used with Salesforce objects,
+so use *Standard Page* when creating your custom community pages. You can start your standard page by choosing one of the preconfigured pages or making a *New Blank Page*.
+
+To make a new blank page, select the layout, then enter the new page's *Name* and *URL*. Use the URL to make
```

Comment:
This is a task hidden a concept/reference topic set. See opening edit statements.
Another reason to separate creating pages from customizing them for branding.

Reply:
True. I rewrote this part as concept info describing the Builder UI. At some future point we may also add a task page to walk a community admin through using these features.